app/services/hmpps_api/oauth/token.rb

```diff
@@ -32,13 +32,15 @@ def valid_token_with_scope?(scope, role: nil)
         return false if payload['scope'].nil?
         return false unless payload['scope'].include? scope
 
-        # For the time being let this through with just a warning log. When we've gathered enough
-        # data and informed all the callers, we'll return false here to enforce the correct role
         if role && !payload.fetch('authorities', []).include?(role)
-          Rails.logger.warn(
-            "event=api_token_missing_role,token_user_name=#{payload['user_name']}," \
-            "token_client_id=#{payload['client_id']}}|API access with missing role #{role}"
+          Rails.logger = Logger.new($stdout) if Rails.env.production?
```

Comment:
I'm very unsure about this... this will affect the logging for the entire application which does it's own "direct to Kibana" thing when running as a web service

Reply:
I shall remove it -- it's an attempt to force the log into Sentry (it hasn't been appearing)

Reply:
Logs don't go into Sentry, they go into Kibana